ops/pebble.py

```diff
@@ -1028,6 +1056,48 @@ class CheckStatus(enum.Enum):
     DOWN = 'down'
 
 
+class LogTarget:
+    """Represents a log target in a Pebble configuration layer."""
+
+    def __init__(self, name: str, raw: Optional['LogTargetDict'] = None):
+        self.name = name
+        dct: LogTargetDict = raw or {}
+        self.override: str = dct.get('override', '')
+        self.type_ = dct.get('type', '')
```

Comment:
I think we should just call this `self.type` (similar to `FileInfo.type`). `type()` is a built-in function, but there's nothing wrong with using it as an instance attribute (which always has the instance/self namespace to keep it separate).